test_validators.py

```diff
@@ -18,6 +19,25 @@ def test_invalid_uuid(uuid):
     assert not verify_uuid_format(uuid)
 
 
+@pytest.mark.parametrize("satellite_id", ["4a8fb994-57fe-4dbb-ad2a-9e922560b6c1", "1000056432"])
+def test_valid_satellite_id(satellite_id):
+    assert verify_satellite_id(satellite_id) is True
+
+
+@pytest.mark.parametrize(
+    "satellite_id",
+    [
+        "123456789",
```

Comment:
```suggestion
        "123456789",
        "12345678901",
```

The first attempt to this pull request did a false positive match for too long IDs. A regression test would come in handy.